src/sparseml/pytorch/models/registry.py

```diff
@@ -91,23 +92,33 @@ def create(
             loaded in model, False otherwise; default True
         :param ignore_error_tensors: tensors to ignore if there are errors in loading
         :param kwargs: any keyword args to supply to the model constructor
-        :return: the instantiated model
+        :return: The instantiated model if key is given else a Tuple containing
+            the instantiated model and the loaded key
         """
-        if key not in ModelRegistry._CONSTRUCTORS:
+        _key = key
+
+        if _key is None:
```

Comment:
is by default key value is `None`, it would be enough to write `if _key:`.

Reply:
is `None` is a safer check since other things could evaluate to be false